src/main/java/com/github/gilday/darkmode/DarkModeSync.java

```diff
@@ -50,7 +53,11 @@ public void dispose() {
 
   private void updateLafIfNecessary() {
     final LookAndFeelInfo current = lafManager.getCurrentLookAndFeel();
-    final boolean isDarkMode = isDarkMode();
+    if (SystemInfo.isMacOSMojave) {
+      isDarkMode = isMacOsDarkMode();
+    } else {
```

Comment:
Let's add a check for Windows with an else that throws if neither macOS Mojave or Windows is detected.

Reply:
I implemented this.